src/Symfony/Component/FeatureFlag/README.md

```diff
@@ -0,0 +1,20 @@
+FeatureFlag Component
+======================
+
+The FeatureFlag component provides a service that checks if a feature is
```

Comment:
You described what and how
Perhaps add the why?
Like « it helps you split code execution flow » or something like that

Reply:
Indeed, the "why" is maybe the first thing to explain :sweat_smile: